Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -55,7 +55,7 @@ public function __construct(SymfonyFixturesLoader $fixturesLoader, ?ManagerRegis
     }
 
     /** @return void */
```

Comment:
This type declaration can now be removed.

```suggestion
```

Reply:
Thanks, Grégoire, for the review and fix you already applied.

Hopefully, the other reviewers can give their thumbs-up soon :-)

WR

D